src/interface/widgets/widget.cpp

```diff
@@ -205,7 +205,7 @@ void Visualizer::drawFieldLines(const world::Field &field, QPainter &painter) {
         bool weAreYellow = SETTINGS.isYellow();
 
         // draw the hint for us
-        auto usGoalLine = FieldComputations::getGoalSides(field, true);
+        LineSegment usGoalLine = FieldComputations::getGoalSides(field, true);
```

Comment:
Why remove `auto`?

Reply:
I am not quite fan of auto, but if you are fan of auto then we can keep it. I think this is a clear example where people can get confused of auto, because the return type of getGoalSides is changed from Line to LineSegment. But now it looks like it always returned a LineSegment, since the return type Line was masked by auto.